packages/components/src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from "react";
+import type { SystemProp, Theme } from "@xstyled/styled-components";
+import { FileWithPath, useDropzone } from "react-dropzone";
+import forEach from "lodash/forEach";
+import { Text } from "../../primitives/Text";
+import { ProgressBar } from "../ProgressBar";
+import { Box } from "../../primitives/Box";
+import { Icon, IconProps } from "../Icon";
+import { CancelUploadButton } from "./CancelUploadButton";
+
+const MAX_FILE_SIZE = 50 * 1024 * 1024;
```

Comment:
Why the multiplication here?

Reply:
Probably taken from my POC code. Is there an easier and more readable way to get 50 MB in JS?
I guess there is nothing like in Rails where you can have `50.megabytes`.